app/src/main/res/layout/legend_row.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    >
+
+    <ImageView
+        android:id="@+id/fragment_legend_image"
+        android:layout_width="50dp"
+        android:layout_height="50dp"
+        android:src="@drawable/k"
```

Comment:
I'm not sure exactly what you mean.
Should I add the following line:        
`xmlns:tools="http://schemas.android.com/tools"`
Or do you want something else done?